modules/nodes-nodes-rebooting-gracefully.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-rebooting.adoc
+
+[id="nodes-nodes-rebooting-gracefully_{context}"]
+= Rebooting a node gracefully
+
+Before rebooting a node, it is recommended to take a backup of etcd data to avoid any data loss on the node.
+
+.Procedure
+
+To perform a graceful restart of a node:
+
+. Mark the node as unschedulable:
++
+[source,terminal]
+----
+$ oc adm cordon <node1>
+----
++
+. Drain the node to remove all the running pods:
++
+[source,terminal]
+----
+$ oc adm drain <node1>
```

Comment:
OCP needs `--ignore-daemonsets` and `--delete-emptydir-data`, because almost every cluster runs a DaemonSet by now and also EmptyDir volumes are frequently used.

Reply:
It looks like these flags still need to be added to this step:

`$ oc adm drain --ignore-daemonsets --delete-emptydir-data <node1> `

And this would need QE review. Thanks!